.github/workflows/monitor.yml

```diff
@@ -0,0 +1,44 @@
+name: Root Monitor
+
+# Execute this as a daily cron job.
+on:
+  # Enable cron for re-signing snapshot and timestamp every week
+  schedule:
+    - cron: '0 0 */1 * *' # every day
+  workflow_dispatch:
+
+jobs:
+  root-probe:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/setup-go@v2
```

Comment:
Nit: I like to pin GHA to their digest (and use dependabot to monitor for updates).